README.md

```diff
@@ -69,3 +69,55 @@ vault.get_kv_secret(mount_point="example", path="path/test")
    }
 ]
 ```
+
+## Create or Update KV Secret
+Method to create an secret or update an existing one in a given path. 
+
+```python
+vault = VaultProvider()
+secret = {"example": "Test", "number": 42}
+vault.create_or_update_kv_secret(mount_point="example", path="path/test", secret=secret)
+```
+```
+{
+   'request_id': '4ce62ee7-0f88-3efc-d745-5e2fbc423789',
+   'lease_id': '',
+   'renewable': False,
+   'lease_duration': 0,
+   'data': {
```

Comment:
Is this example correct? Looking to the other example in the README this one looks like the "metadata", not the "data".

Reply:
That's correct, the `create_or_update` and` patch` methods doesn't return a "metadata" key

Reply:
Ok, thanks for confirming